app.json

```diff
@@ -9,11 +9,11 @@
     },
     "RACK_ENV": {
       "required": true,
-      "value": "production"
+      "value": "review"
     },
     "RAILS_ENV": {
       "required": true,
-      "value": "production"
+      "value": "review"
```

Comment:
Is this the way to set env variable for heroku?

Reply:
Yeah. Just go into settings for the review app, then change the environment variable manually